paper_trail.gemspec

```diff
@@ -19,9 +19,7 @@ has been destroyed.
   s.email = "jared@jaredbeck.com"
   s.license = "MIT"
 
-  s.files = `git ls-files -z`.split("\x0").select { |f|
-    f.match(%r{^(Gemfile|LICENSE|lib/|paper_trail.gemspec)})
-  }
+  s.files = Dir.glob(["CHANGELOG.md", "doc/warning_about_not_setting_whodunnit.md", "lib/**/*", "LICENSE", "README.md"])
```

Comment:
Aw, this now breaks the LineLength issue. I believe updating RuboCop simply should be enough.

OTOH, as I see it, the `README.md` is absolutely well written and I don't see the need to include `doc/`, do you think we can get rid of it altogether?

Either way, I don't mind, really and also, thanks a bunch for your work on this! ❤️

Reply:
If it helps, we can put array entries on their own lines.  I think it could more flexible to include the doc directory instead of a single file within it in case something changes.
```
s.files = Dir.glob([
  "CHANGELOG.md",
  "doc/*",
  "lib/**/*",
  "LICENSE",
  "README.md"
])
```

Reply:
Sure, looks good! \o/